### DISCUSSION TITLE
Please write a small getting started

### DISCUSSION THREAD

**Discussion by wozhdal** [yume-chan/ya-webadb#422] [General]
Hi,

i am very interested by this project but i am a noob to node.
Yes i know, i must take the hard path to learn ;)

but can someone juste write 4-5 lines on a "getting started" ?

- i gitted cloned
- cd to apps/demo
- npm install --force (because npm install is not happy with typescript version dependencies)
and got
`
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@yume-chan%2fadb-backend-direct-sockets - Not found
npm ERR! 404
npm ERR! 404  '@yume-chan/adb-backend-direct-sockets@^0.0.9' is not in this registry.
`
i understand that there are local libraries but i am not sure how to access it.
I tried :
- some npm install -g in the folder of the adb-backend-direct socket
- copy the folder from libraries to node_modules
- modify package.json of demo to use local path  (../../libraries/adb-backend/..=
- modify connect.tx and import lines to use local paths

**Comment by yume-chan** [comment]
See https://github.com/yume-chan/ya-webadb/blob/master/CONTRIBUTE.md

**Reply by wozhdal** [reply]
perfect. Thank you :)

**Reply by Naxexia** [reply]
Now the page shows 404...
